src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -132,54 +136,15 @@ public void writeToNBT(@NotNull final NBTTagCompound compound)
         @NotNull final NBTTagList saplingsLeavesTagList =
                 leavesToSaplingMap.entrySet().stream().map(entry ->  writeLeaveSaplingEntryToNBT(entry.getKey(), entry.getValue())).collect(NBTUtils.toNBTTagList());
         compound.setTag(TAG_SAP_LEAVE, saplingsLeavesTagList);
-
-        @NotNull final NBTTagList saplingTagList =
-                saplings.stream().map(sap ->  sap.getItemStack().writeToNBT(new NBTTagCompound())).collect(NBTUtils.toNBTTagList());
-        compound.setTag(TAG_SAPLINGS, saplingTagList);
-
-        try
-        {
-            @NotNull final NBTTagList oresTagList = ores.stream().map(ore -> NBTUtil.writeBlockState(new NBTTagCompound(), ore.getDefaultState())).collect(NBTUtils.toNBTTagList());
-            compound.setTag(TAG_ORES, oresTagList);
-        }
-        catch (final Exception e)
-        {
-            Log.getLogger().error("Error caught during ore serialization!", e);
-        }
     }
 
     @Override
     public void readFromNBT(@NotNull final NBTTagCompound compound)
     {
         NBTUtils.streamCompound(compound.getTagList(TAG_SAP_LEAVE, Constants.NBT.TAG_COMPOUND))
-                .map(CompatabilityManager::readLeaveSaplingEntryFromNBT)
+                .map(CompatibilityManager::readLeaveSaplingEntryFromNBT)
```

Comment:
If we now discover on creation why still reading / writing?

Reply:
The saplings and leaves map is populated from the lumberjacks so we still need to save it.